app/src/main/java/com/android/sample/ui/activity/CreateActivity.kt

```diff
@@ -539,7 +539,7 @@ fun CreateActivityScreen(
                               "You must be logged in to create an activity.",
                               Toast.LENGTH_SHORT)
                           .show()
-                    } else if (hourDateViewModel.isBeginGreaterThanEnd(startTime, duration)) {
```

Comment:
checks if start time is before the end time. (if not an error toast will be displayed in the screen with the message: "The start time must be before the end time.".